include/rapid_pbd/action_names.h

```diff
@@ -6,6 +6,7 @@
 
 namespace rapid {
 namespace pbd {
+// will be preprended with /rapid_pbd/
```

Comment:
Change comment to say `// Names without a leading slash will be prepended with /rapid_pbd/ in the launch files.`